tests/test_analysis_tools.py

```diff
@@ -1,27 +1,21 @@
 import os
-from io import StringIO
 from unittest.mock import MagicMock, mock_open, patch
 
-import MDAnalysis as mda
-import numpy as np
 import pytest
 
 from mdagent.tools.base_tools import VisFunctions
 from mdagent.tools.base_tools.analysis_tools.plot_tools import PlottingTools
 from mdagent.tools.base_tools.analysis_tools.ppi_tools import ppi_distance
-from mdagent.tools.base_tools.analysis_tools.rmsd_tools import RMSDFunctions
+from mdagent.tools.base_tools.analysis_tools.rmsd_tools import lprmsd, rmsd, rmsf
+
+#################### PLOTTING ####################
```

Comment:
Instead, these should just be separate files

Reply:
header removed. see my other comment about visualization & plotting tests